tests/environment-build.bats

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test that we can build environments
+#
+#
+# ---------------------------------------------------------------------------- #
+
+load test_support.bash;
+
+# bats file_tags=activate,init
+
+
+# ---------------------------------------------------------------------------- #
+
+setup_file() {
+  common_file_setup;
+  export FLOX_FEATURES_ENV="rust";
```

Comment:
note nonblocking: I think we should be able to drop that everywhere

Reply:
I only remembered to drop it in this particular file. If I need subsequent commits I'll drop it in those.